nx.json

```diff
@@ -27,12 +27,18 @@
     "barista-tools": {
       "tags": ["type:tooling"]
     },
+    "examples-tools": {
+      "tags": ["type:tooling"]
```

Comment:
please add implicit dependency so when you change something in the tools all the examples and demos are affected:
```json
"implicitDependencies": ["demos", "examples"]
```

Reply:
you can test the dependency graph with `nx affected:dep-graph --file=/path/to/tooling/file.ts`